Acme.Interface.WebAPI/Dockerfile

```diff
@@ -0,0 +1,36 @@
+#See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+FROM mcr.microsoft.com/dotnet/aspnet:8.0-alpine AS base
+USER app
+WORKDIR /app
+
+EXPOSE 8080
+EXPOSE 8081
+
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
+ARG BUILD_CONFIGURATION=Release
+WORKDIR /src
+
+# copy all the layers' cyproj files into their respective folders
+COPY ["Acme.Application/Acme.Application.csproj", "Acme.Application/"]
+COPY ["Acme.Core/Acme.Core.csproj", "Acme.Core/"]
+COPY ["Acme.Infrastructure.EF.PostgreSql/Acme.Infrastructure.EF.PostgreSql.csproj", "Acme.Infrastructure.EF.PostgreSql/"]
+COPY ["Acme.Interface.WebAPI/Acme.Interface.WebAPI.csproj", "Acme.Interface.WebAPI/"]
+COPY ["Enterwell.Exceptions.Standard/Enterwell.Exceptions.Standard.csproj", "Enterwell.Exceptions.Standard/"]
+COPY ["Enterwell.Exceptions.Standard.Web/Enterwell.Exceptions.Standard.Web.csproj", "Enterwell.Exceptions.Standard.Web/"]
+
+# run restore over the WebAPI.Starter project - this pulls restore over all other dependent projects as well
+RUN dotnet restore "./Acme.Interface.WebAPI/./Acme.Interface.WebAPI.csproj"
```

Comment:
Think I commented on this in [another PR's](https://github.com/Enterwell/dotnet-starter/pull/12#discussion_r1455837653) comment